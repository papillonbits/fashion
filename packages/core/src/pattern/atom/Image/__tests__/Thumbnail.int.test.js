import { newspaper, like, share, check, cross, printer } from './Thumbnail.int.story'

describe('<Thumbnail />', () => {
  describe('Render', () => {
    test('must match newspaper', () => {
      expect(global.renderToJSON(newspaper())).toMatchSnapshot()
    })

    test('must match like', () => {
      expect(global.renderToJSON(like())).toMatchSnapshot()
    })

    test('must match share', () => {
      expect(global.renderToJSON(share())).toMatchSnapshot()
    })

    test('must match check', () => {
      expect(global.renderToJSON(check())).toMatchSnapshot()
    })

    test('must match cross', () => {
      expect(global.renderToJSON(cross())).toMatchSnapshot()
    })

    test('must match printer', () => {
      expect(global.renderToJSON(printer())).toMatchSnapshot()
    })
  })
})
