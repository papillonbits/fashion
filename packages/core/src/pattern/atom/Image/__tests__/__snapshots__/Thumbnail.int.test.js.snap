// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Thumbnail /> Render must match check 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M896 213.333q18.333 0 30.5 12.167t12.167 30.5q0 18-12.333 30.333l-512 512q-12.333 12.333-30.333 12.333t-30.333-12.333l-256-256q-12.333-12.333-12.333-30.333 0-18.333 12.167-30.5t30.5-12.167q18 0 30.333 12.333l225.667 226 481.667-482q12.333-12.333 30.333-12.333z"
      />
    </svg>
  </div>
</div>
`;

exports[`<Thumbnail /> Render must match cross 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M810.667 170.667q18.333 0 30.5 12.167t12.167 30.5q0 18-12.333 30.333l-268.667 268.333 268.667 268.333q12.333 12.333 12.333 30.333 0 18.333-12.167 30.5t-30.5 12.167q-18 0-30.333-12.333l-268.333-268.667-268.333 268.667q-12.333 12.333-30.333 12.333-18.333 0-30.5-12.167t-12.167-30.5q0-18 12.333-30.333l268.667-268.333-268.667-268.333q-12.333-12.333-12.333-30.333 0-18.333 12.167-30.5t30.5-12.167q18 0 30.333 12.333l268.333 268.667 268.333-268.667q12.333-12.333 30.333-12.333z"
      />
    </svg>
  </div>
</div>
`;

exports[`<Thumbnail /> Render must match like 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M755.188 64c-107.63 0-200.258 87.554-243.164 179-42.938-91.444-135.578-179-243.216-179-148.382 0-268.808 120.44-268.808 268.832 0 301.846 304.5 380.994 512.022 679.418 196.154-296.576 511.978-387.206 511.978-679.418 0-148.392-120.43-268.832-268.812-268.832z"
      />
    </svg>
  </div>
</div>
`;

exports[`<Thumbnail /> Render must match newspaper 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M896 256v-128h-896v704c0 35.346 28.654 64 64 64h864c53.022 0 96-42.978 96-96v-544h-128zM832 832h-768v-640h768v640zM128 320h640v64h-640zM512 448h256v64h-256zM512 576h256v64h-256zM512 704h192v64h-192zM128 448h320v320h-320z"
      />
    </svg>
  </div>
</div>
`;

exports[`<Thumbnail /> Render must match printer 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M256 64h512v128h-512v-128z M960 256h-896c-35.2 0-64 28.8-64 64v320c0 35.2 28.794 64 64 64h192v256h512v-256h192c35.2 0 64-28.8 64-64v-320c0-35.2-28.8-64-64-64zM128 448c-35.346 0-64-28.654-64-64s28.654-64 64-64 64 28.654 64 64-28.652 64-64 64zM704 896h-384v-320h384v320z"
      />
    </svg>
  </div>
</div>
`;

exports[`<Thumbnail /> Render must match share 1`] = `
<div>
  <div>
    <svg
      class="thumbnail"
      fill=""
      font-size="1px"
      height="30"
      viewBox="0 0 1024 1024"
      width="30"
    >
      <path
        d="M864 704c-45.16 0-85.92 18.738-115.012 48.83l-431.004-215.502c1.314-8.252 2.016-16.706 2.016-25.328s-0.702-17.076-2.016-25.326l431.004-215.502c29.092 30.090 69.852 48.828 115.012 48.828 88.366 0 160-71.634 160-160s-71.634-160-160-160-160 71.634-160 160c0 8.622 0.704 17.076 2.016 25.326l-431.004 215.504c-29.092-30.090-69.852-48.83-115.012-48.83-88.366 0-160 71.636-160 160 0 88.368 71.634 160 160 160 45.16 0 85.92-18.738 115.012-48.828l431.004 215.502c-1.312 8.25-2.016 16.704-2.016 25.326 0 88.368 71.634 160 160 160s160-71.632 160-160c0-88.364-71.634-160-160-160z"
      />
    </svg>
  </div>
</div>
`;
